### PR TITLE
Create Bar graph

### PR DESCRIPTION
**Pull request by Naomi-Dennis**

### CODE REVIEW

**Comment by damonkelley** [comment]
@Naomi-Dennis Could you include a screenshot in the description!? 😄

**Comment by Naomi-Dennis** [comment]
[Bar Graph Preview](https://drive.google.com/file/d/1RqXJ8YQ9qRIcifgN2MHJyTn0p5aGmVdc/view?usp=sharing)

**Comment by damonkelley** [comment]
Thanks! BTW, you can also attach images directly to the PR and it will display in the comment or description :smile:

Mind the Gapp Graph Preview

**Comment by damonkelley** [comment]
We haven't discussed any styling yet, but I envisioned that the bars in the graph might be green to match the G train bullet. https://en.wikipedia.org/wiki/File:NYCS-bull-trans-G-Std.svg

Is that possible?

File: src/Main.elm [hunk 1]
```diff
@@ -1,56 +1,60 @@
...
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    ]
+    """{"day": ["7 Days Before", "6 Days Before", "5 Days Before", "4 Days BeforE", "3 Days Before", "2 Days Before", "1 Day Before", "Today"],"meanArrivalDifferences": [2.34, -0.10, 5.30, 7.20, -5.32, -4.20, 0.00, 0.32]}"""
```
Comment (damonkelley) [inline src/Main.elm]: This is a little hard to read. Is it possible to reformat it with line breaks? 
```suggestion
    """{
      "day": [
         "7 Days Before",
         "6 Days Before",
         "5 Days Before",
         "4 Days BeforE",
         "3 Days Before",
         "2 Days Before",
         "1 Day Before",
         "Today"
         ],
       "meanArrivalDifferences": [
          2.34,
          -0.10,
          5.30, 
          7.20, 
          -5.32, 
          -4.20, 
          0.00, 
          0.32
         ]
       }"""
```

**Comment by Naomi-Dennis** [comment]
Updated graph screenshot. 

Screen Shot 2020-03-10 at 1 08 20 PM

**Pull request merged**